Makefile

```diff
@@ -270,7 +270,7 @@ gofmt:
 
 govet:
 	@$(ECHO_CHECK) vetting all GOFILES...
-	$(QUIET)$(GO) tool vet api pkg test $(SUBDIRS)
```

Comment:
Please do two things:
* Comment out the preceding line as well
* Create an issue to track re-enabling of `go vet` with go 1.12.